### ISSUE TITLE
As a user, I want my email list to autocomplete so that I don't need to remember entire emails

### ISSUE THREAD

**Issue by Zandrr** [BeTandem/scheduler_frontend#84]
like gmail's bubble up thingy!

**Comment by sthoma** [comment]
https://developers.google.com/people/v1/requests

**Comment by thejinxters** [comment]
I don't think so. It sounds weird to route the the app call that is not reliant on anything else through the backend so we can hit a single app endpoint. I think it can be made directly from the front end.... Unless it requires our api secret key

**Comment by sthoma** [comment]
Second sentence up there was more of a question haha. Looked into it and realized it's actually super straightforward. I think the only downside is adding an extra dependency of googleapis on the frontend

**Comment by thejinxters** [comment]
@sthoma @Zandrr Contact autocomplete can't be done with google's current api. The above link for People only returns the contact names, but does not include emails... making it useless for what we are trying to do. The best we could do is save a list of users -> email addresses that they have previously scheduled and return that for the front-end upon request.
